test/suggestion_engine/server/index.test.js

```diff
@@ -63,6 +63,24 @@ test('should call appropriate activation methods', async () => {
   expect(browser.tabs.update.calledOnce).toEqual(true);
 });
 
+test('should focus bookmark and history tabs if already open', async () => {
+  browser.flush();
+  browser.tabs.query.returns(Promise.resolve([{ id: '1', windowId: '1' }]));
```

Comment:
Minor thing but I think there is a browser.tabs.query.yields API that lets you avoid having to use the Promise API directly. Might be wrong about this but I remember seeing it before.

Reply:
Looks like `stub.resolves` is the correct API. Update incoming.